libraries/interop-ribs/src/main/kotlin/com/bumble/appyx/interop/ribs/InteropBuilder.kt

```diff
@@ -30,6 +32,7 @@ class InteropBuilder<N : Node>(
                     customisations = buildParams.buildContext.customisations
                 )
             )
+            .apply { integrationPoint = this@InteropBuilder.integrationPoint }
```

Comment:
Adding this into BuildContext or the nodeFactory seems potentially safer.

Reply:
`NodeFactory` can be used only in composable context, we can't use it here. `BuildContext`, I don't see how putting it inside will fix it, you still need to get it from somewhere. Parent's has `RIBs'` `buildContext` and it would have `RIBs'` `IntegrationPoint`

Reply:
fair enough